frontend/lib/identification/card/id_card.dart

```diff
@@ -1,27 +0,0 @@
-import 'package:flutter/material.dart';
```

Comment:
This file was moved to `identification/id_card/id_card.dart` and slightly adjusted